### ISSUE TITLE
Wrong generation typeName and mock functions names

### ISSUE THREAD

**Issue by vitorcamachoo** [ardeois/graphql-codegen-typescript-mock-data#40]
Hii,

I'm getting wrong typeNames when reusing types from another generated files and for the functions that are being reused inside mock generated file like you can check in the screenshot:
Configuration:
![Screenshot 2021-02-10 at 11 46 05](https://user-images.githubusercontent.com/20595956/107506124-8fb16980-6b95-11eb-8519-798bddbbf453.png)

Result:
![Screenshot 2021-02-10 at 11 46 17](https://user-images.githubusercontent.com/20595956/107506142-96d87780-6b95-11eb-9dfc-34544a28ae14.png)

**Comment by ardeois** [comment]
Hi,

It's probably an issue with `typenames: pascal-case`, can you try without this option?
If it doesn't work, can you check if it's due to `prefix` option as well?

PRs are welcomed to fix this bug, but if you can't, can you provide a unit test or repository to reproduce ?

Thanks

**Comment by vitorcamachoo** [comment]
I've tried with multiple configurations. The first thing I'm seeing is the top import when I'm trying to reuse types, I have my types on PascalCase and it's importing as lowercase. The order thing is the reuse of the factory functions, for example `mockUser` is defined, but on the rest of the other factories that will use this `mockUser` it is being written as `mockuser` as lowercase.

**Comment by ardeois** [comment]
@vitorcamachoo I think it's because your `typenames` config is wrong
It should be:
```
plugins:
      - typescript-mock-data
config:
      typesFile: ./types.ts'
      typenames: 'pascal-case#pascalCase'
      prefix: 'mock'
```

**Comment by vitorcamachoo** [comment]
Hi @ardeois,
with the configuration that you wrote, I get the same errors like you can see on the attached screenshots:

![Screenshot 2021-02-11 at 22 10 43](https://user-images.githubusercontent.com/20595956/107704981-fe85e400-6cb5-11eb-9336-6b4491ff8544.png)
![Screenshot 2021-02-11 at 22 10 59](https://user-images.githubusercontent.com/20595956/107705003-06de1f00-6cb6-11eb-8019-4dce5466df57.png)
![Screenshot 2021-02-11 at 22 11 27](https://user-images.githubusercontent.com/20595956/107705054-18272b80-6cb6-11eb-8805-db724cef13ae.png)

**Comment by ardeois** [comment]
Could you provide me a repo to reproduce? I tried to reproduce in local and was not able to reproduce.

**Comment by ardeois** [comment]
Fixed in version [1.5.8](https://github.com/ardeois/graphql-codegen-typescript-mock-data/releases/tag/v1.5.8)
